docs/source/user_guide/napari_plugin.md

```diff
@@ -0,0 +1,161 @@
+(target-napari-plugin)=
+# The napari plugin
+
+To enable the interactive visualisation of motion tracks in
+`movement`, we have created an experimental plugin for [napari](napari:).
+Currently, the plugin supports loading 2D
+[poses datasets](target-poses-and-bboxes-dataset), and visualising them
+as points overlaid on video frames.
+
+:::{warning}
+This plugin is still in early stages of development and offers
+limited functionality. We are working on ironing out the kinks and
+gradually adding more features. [Get in touch](target-get-in-touch)
+if you find any bugs or have suggestions for improvements!
+:::
+
+## Installation
+
+The `napari` plugin is shipped with the `movement` package starting from
+version `0.1.0`. If you install `movement` via `conda`, `napari` is already
+included as a dependency. If you use the `pip` installer, make sure to
+install `movement` with the `[napari]` extra:
+
+```sh
+pip install movement[napari]
+```
+
+## Launch the plugin
+
+Type the following command in your terminal:
+
+```sh
+napari -w movement
+```
+
+This will open the `napari` window with the `movement` plugin docked on the
+right-hand side.
+
```

Comment:
Maybe just before this section we can introduce the concept of layer and the layer list. Something like:
```suggestion
In napari, data is typically loaded into layers, which can be reordered and toggled for visibility in the layers list panel. For example, keypoint data can be added as a 'points' layer, while images or videos can be loaded as 'image' layers. Below, we’ll explain how to do this.
```

Reply:
It may be important to say the order affects how the data is presented (e.g., if the video is the top layer it covers the keypoints). 

We may also want to mention one can load multiple videos and pose data files, and toggle their visibility. Maybe people use that?

Reply:
I added the text you suggested, which is already very helpful. I also cross-referenced the word "layers" to napari's [layers at a glance](https://napari.org/0.5.3/guides/layers.html) guide.